### ISSUE TITLE
Request more detailed experimental parameters

### ISSUE THREAD

**Issue by PKQZPC** [xmed-lab/GenericSSL#9]
Hi,

Thank you very much for your excellent work. I would like to ask for a guide on using the dataset mentioned in your paper. I have attempted multiple rounds of training on the Synapse dataset and tried different epochs, but found that the accuracy of the reproduced results still differs from the data in your paper. May I ask if it is possible to provide more detailed parameter explanations so that I can better reproduce your results?

Thank you very much for your help!

**Comment by McGregorWwww** [comment]
Hi, thank you for your interest of our work! We've trained the model for all tasks consistently over 300 epochs. To adapt the learning rates for different tasks, kindly refer to Table 8:
<img width="882" alt="image" src="https://github.com/xmed-lab/GenericSSL/assets/35693095/4d6ddb14-3195-498d-b258-2dad59c8465b">
Hope this helps.

**Comment by shuaiaihang** [comment]
Hi,
Thanks for your remarkable work. I've conducted experiments on the LA dataset using the configuration consistent with Table 8 in your paper (consistent with the config file). The obtained results are 78.1 (dice, 300 epochs) and 81.2 (dice, 200 epochs). Could you please advise if there are any additional parameters I should adjust to better replicate your reported results on the LA dataset?

Thank you very much for your assistance.

**Comment by shuaiaihang** [comment]
Hi,
Thank you very much your remarkable work. I've conducted experiments on the LA dataset using the configuration consistent with Table 8 in your paper (same with the config file). The obtained results show some variance from those reported in the paper. Could you please advise if there are any additional parameters I should adjust to better replicate your reported results on the LA dataset?

Thank you very much for your assistance.
